lms/djangoapps/instructor/views/api.py

```diff
@@ -1125,6 +1145,7 @@ def list_course_problems(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@require_post_params(group_id="id of group to rename", group_name="the new name of the group query with id group_id")
```

Comment:
My bad, per [Steven's comment](https://github.com/Stanford-Online/edx-platform/pull/268#discussion_r28472651) can we change the formatting to:
```
@require_post_params(
    group_id="id of group to rename",
    group_name="the new name of the group query with id group_id",
)
```